packages/@textlint/fixer-formatter/package.json

```diff
@@ -39,11 +39,11 @@
   "dependencies": {
     "@textlint/module-interop": "^12.1.0",
     "@textlint/types": "^12.1.0",
-    "chalk": "^1.1.3",
+    "chalk": "^4.1.2",
     "debug": "^4.3.3",
     "diff": "^4.0.2",
     "is-file": "^1.0.0",
-    "string-width": "^1.0.2",
+    "string-width": "^4.2.3",
```

Comment:
Confirm: No API changes.
https://github.com/sindresorhus/string-width/compare/v1.0.2...v2.0.0
https://github.com/sindresorhus/string-width/releases/tag/v3.0.0
https://github.com/sindresorhus/string-width/releases/tag/v4.0.0

Reply:
`string-width` do not have to update.
Therefore, `string-width` is excluded from the update target: https://github.com/textlint/textlint/pull/842/commits/902c362af39f019a0170d2d384a3f3f38e9bbceb

Reply:
Will it be separated PR?

I'm ok that a single PR includes multiple package updates.

Reply:
After check again, it seems that it is not necessary to update `string-width` to update `ansi-regex`.
Therefore, it was excluded from the update target.

Reply:
OK.
It seems like separated issue.
I've created it as new issue.
https://github.com/textlint/textlint/issues/843